src/components/ui/Dialog.tsx

```diff
@@ -0,0 +1,103 @@
+/**
+ * This file is based on the Dialog component from shadcn and customized for our needs.
+ *
+ * See the official docs for more info:
+ * shadcn/ui: https://ui.shadcn.com/docs/components/dialog
+ * Radix UI: https://www.radix-ui.com/themes/docs/components/dialog
+ */
+
+'use client';
+
+import { Icon } from '@/components/ui';
+import { cn } from '@/lib/tailwind/utils';
+
+import * as DialogPrimitive from '@radix-ui/react-dialog';
+import * as React from 'react';
```

Comment:
Wildcard import should not be used if possible.
[Reference](https://stackoverflow.com/a/43569378)

I propose something like this.
```
import {
  Close,
  Content as DialogPrimitiveContent,
  Description as DialogPrimitiveDescription,
  Overlay as DialogPrimitiveOverlay,
  Portal,
  Title as DialogPrimitiveTitle,
} from '@radix-ui/react-dialog';
import { ComponentPropsWithoutRef, ElementRef, forwardRef, HTMLAttributes } from 'react';
```

Reply:
> `import * as React from 'react';`
I believe this is OK because this is a very general style to import `React`. What do you think?

> import * as DialogPrimitive from '@radix-ui/react-dialog';
This is the default code when I generated the original Dialog component. I prefer always to use name-import, too. Do you still think it is better to modify the default state?

Reply:
It concluded in not using wildcard import everywhere.